src/subscriptions/subscriptionReaper.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+import { DynamoDbDataService, DynamoDb } from 'fhir-works-on-aws-persistence-ddb';
+
+const enableMultiTenancy = process.env.ENABLE_MULTI_TENANCY === 'true';
+const dbService = new DynamoDbDataService(DynamoDb, false, {
+    enableMultiTenancy,
```

Comment:
Now that I think of it, we actually need 2 `DynamoDbDataService`. The reaper needs a ddb without `enableMultiTenancy` since it will be deleting subscriptions for all tenants so it must be able to fetch all subscriptions with `getActiveSubscriptions`. 
But a ddb with `enableMultiTenancy` is needed to call `dbService.deleteResource` in order to deleet tenant-specific resources

Reply:
That makes sense, I added another dbService and handled multitenant deployments as you commented below.